src/cursor/cur_backup_incr.c

```diff
@@ -93,6 +94,7 @@ __curbackup_incr_blkmod(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_CURSOR_BAC
             WT_ERR(__wt_backup_load_incr(session, &b, &cb->bitstring, cb->nbits));
             cb->bit_offset = 0;
             F_SET(cb, WT_CURBACKUP_INCR_INIT);
+            __wt_verbose(session, WT_VERB_BACKUP, "Modified incr block config: \"%s\"", config);
```

Comment:
I think the real fix can be here. As in, The code this is part of could be within the else clause where we clear `WT_CURBACKUP_RENAME`. That way we don't set the `WT_CURBACK_INCR_INIT` flag nor load the blocks in the case of rename. It would look like:
```
        WT_ERR_NOTFOUND_OK(__wt_config_subgets(session, &v, "rename", &b), true);
        if (ret == 0 && b.val)
            F_SET(cb, WT_CURBACKUP_RENAME);
        else {
            F_CLR(cb, WT_CURBACKUP_RENAME);
            /*
             * We found a match. Load the block information into the cursor.
             */
            if ((ret = __wt_config_subgets(session, &v, "blocks", &b)) == 0) {
                WT_ERR(__wt_backup_load_incr(session, &b, &cb->bitstring, cb->nbits));
               cb->bit_offset = 0;
                F_SET(cb, WT_CURBACKUP_INCR_INIT);
            }
        }
```

Reply:
Thanks! This method works nicely